Frameworks/CoreText/CTLine.mm

```diff
@@ -322,8 +347,7 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
  @Status Interoperable
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
-    _CTLine* line = static_cast<_CTLine*>(lineRef);
-
+    __CTLine* line = const_cast<__CTLine*>(lineRef);
```

Comment:
ah okay , you want to move the whole function into __CTLine.
uh, this is where it's going to get weird. __CTline is already violated elsewhere, so it has to go public strut.
Then this moves into it and expose a function. it would be ideal to clean everything up, so please open a bug for it.
The main purpose of this is to get rid of foundation.

---
In reply to: [112742309](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112742309) [](ancestors = 112742309)